tests/static/locators.html

```diff
@@ -24,6 +27,9 @@
         document.querySelector('#input').addEventListener(
             'change', e => { window.check = event.currentTarget.checked; }, false
         );
+        document.querySelector('#checkedInput').addEventListener(
+            'change', e => { window.uncheck = !event.currentTarget.checked; }, false
+        );
```

Comment:
Could you merge the `Check` and `Uncheck` tests into one, so that we use the same `checkbox` element and `window.check` property for both tests?

Reply:
Thanks, makes sense 🙏

https://github.com/grafana/xk6-browser/pull/365/commits/8e7f57b509a7dd3e3e07996595f34bd9189c83d5